### ISSUE TITLE
Editing Internship

### ISSUE THREAD

**Issue by JD499** [pittcsc/Summer2023-Internships#2756]
### Link to Internship Posting

https://www.geckorobotics.com/careers/apply?gh_jid=5695257003

### Company Name

Gecko Robotics

### Internship Title

Software Engineering Internship - Summer 2024

### Location

Boston, MA

### What term(s) is this internship offered for?

Summer 2024

### Does this internship offer sponsorship?

None

### Is this internship currently accepting applications?

None

### Remove this internship from the list?

- [X] Yes, remove this internship

### Email associated with your GitHub account (Optional)

jd499b@gmail.com

### Reason for edit

link leads to "Sorry, but we can't find that page."

**Comment by JD499** [comment]
> There was an error with our github action. Error - We could not find this internship in our list. Please double check you inserted the right url

updated url

**Comment by JD499** [comment]
Hello @cwcoogan,

I noticed that the action could not be processed earlier due to an incorrect URL. I have now updated the URL to match precisely with the one provided in `listings.json`, which should resolve the problem.

Additionally, this correction aligns with the issues #2754 and #2755, where similar inconsistencies occurred. I believe this update should help in addressing the concerns raised in these issues as well.

Looking forward to your feedback or any further steps needed to ensure the accuracy and functionality of the action.

Thank you!
